data/processed/hesa/total_university_buildings.csv

```diff
@@ -0,0 +1,121 @@
+year,nuts_id,nuts_year_spec,total_university_buildings
+2015,UKH2,2018,419.0
+2016,UKH2,2018,413.0
+2017,UKH2,2018,407.0
+2015,UKJ1,2018,684.0
+2016,UKJ1,2018,661.0
+2017,UKJ1,2018,665.0
+2015,UKD6,2018,251.0
+2016,UKD6,2018,213.0
+2017,UKD6,2018,213.0
+2015,UKK3,2018,72.0
+2016,UKK3,2018,82.0
+2017,UKK3,2018,85.0
+2015,UKD1,2018,94.0
+2016,UKD1,2018,96.0
+2017,UKD1,2018,98.0
+2015,UKF1,2018,516.0
+2016,UKF1,2018,503.0
+2017,UKF1,2018,519.0
+2015,UKK4,2018,369.0
+2016,UKK4,2018,368.0
+2017,UKK4,2018,368.0
+2015,UKK2,2018,42.0
+2016,UKK2,2018,42.0
+2017,UKK2,2018,40.0
+2015,UKH1,2018,475.0
+2016,UKH1,2018,473.0
+2017,UKH1,2018,484.0
+2015,UKL2,2018,368.0
+2016,UKL2,2018,372.0
+2017,UKL2,2018,375.0
+2015,UKE1,2018,299.0
+2016,UKE1,2018,187.0
+2017,UKE1,2018,185.0
+2015,UKM7,2018,1129.0
+2016,UKM7,2018,1126.0
+2017,UKM7,2018,1124.0
+2015,UKH3,2018,422.0
+2016,UKH3,2018,406.0
+2017,UKH3,2018,403.0
+2015,UKK1,2018,833.0
+2016,UKK1,2018,876.0
+2017,UKK1,2018,907.0
+2015,UKD3,2018,332.0
+2016,UKD3,2018,321.0
+2017,UKD3,2018,318.0
+2015,UKJ3,2018,510.0
+2016,UKJ3,2018,517.0
+2017,UKJ3,2018,600.0
+2015,UKG1,2018,60.0
+2016,UKG1,2018,62.0
+2017,UKG1,2018,64.0
+2015,UKM6,2018,111.0
+2016,UKM6,2018,113.0
+2017,UKM6,2018,116.0
+2015,UKI4,2018,260.0
+2016,UKI4,2018,265.0
+2017,UKI4,2018,266.0
+2015,UKI3,2018,777.0
+2016,UKI3,2018,815.0
+2017,UKI3,2018,846.0
+2015,UKJ4,2018,578.0
+2016,UKJ4,2018,567.0
+2017,UKJ4,2018,541.0
+2015,UKD4,2018,416.0
+2016,UKD4,2018,412.0
+2017,UKD4,2018,440.0
+2015,UKF2,2018,548.0
+2016,UKF2,2018,538.0
+2017,UKF2,2018,454.0
+2015,UKF3,2018,82.0
+2016,UKF3,2018,86.0
+2017,UKF3,2018,89.0
+2015,UKD7,2018,311.0
+2016,UKD7,2018,304.0
+2017,UKD7,2018,307.0
+2015,UKM5,2018,180.0
+2016,UKM5,2018,145.0
+2017,UKM5,2018,146.0
+2015,UKE2,2018,313.0
+2016,UKE2,2018,317.0
+2017,UKE2,2018,373.0
+2015,UKN0,2018,333.0
+2016,UKN0,2018,334.0
+2017,UKN0,2018,318.0
+2015,UKC2,2018,361.0
+2016,UKC2,2018,348.0
+2017,UKC2,2018,335.0
+2015,UKI5,2018,61.0
+2016,UKI5,2018,62.0
+2017,UKI5,2018,64.0
+2015,UKI6,2018,167.0
+2016,UKI6,2018,168.0
+2017,UKI6,2018,169.0
+2015,UKI7,2018,240.0
+2016,UKI7,2018,238.0
+2017,UKI7,2018,239.0
+2015,UKG2,2018,390.0
+2016,UKG2,2018,274.0
+2017,UKG2,2018,358.0
+2015,UKE3,2018,410.0
+2016,UKE3,2018,421.0
+2017,UKE3,2018,413.0
+2015,UKJ2,2018,856.0
+2016,UKJ2,2018,822.0
+2017,UKJ2,2018,848.0
+2015,UKC1,2018,407.0
+2016,UKC1,2018,406.0
+2017,UKC1,2018,408.0
+2015,UKM8,2018,439.0
+2016,UKM8,2018,471.0
+2017,UKM8,2018,470.0
+2015,UKG3,2018,865.0
+2016,UKG3,2018,864.0
+2017,UKG3,2018,956.0
+2015,UKL1,2018,719.0
+2016,UKL1,2018,710.0
+2017,UKL1,2018,716.0
+2015,UKE4,2018,519.0
```

Comment:
In the schema the value is marked as an `int` but values are floats

Reply:
Now integer